### PR TITLE
This is a WIP branch, on which I try to understand the deployment by creating a Vagrantfile

### PR DESCRIPTION
**Pull request by maxhbr**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+# see: https://oss-augur.readthedocs.io/en/master/getting-started/installation.html
+python3 -m venv $HOME/.virtualenvs/augur_env
+source $HOME/.virtualenvs/augur_env/bin/activate
+pip install wheel
+
+cd /vagrant
+python setup.py bdist_wheel
+make clean
+pip install .
+make install
```
Comment (maxhbr) [inline Vagrantfile:49]: This line currently fails with

```
    default: **********************************
    default: Installing message_insights_worker...
    default: **********************************
    default: Processing /vagrant/workers/message_insights_worker
    default: Requirement already satisfied: Flask-Cors==3.0.10 in /root/.virtualenvs/augur_env/lib/python3.8/site-packages (from message-insights-worker==0.0.0) (3.0.10)
    default: Requirement already satisfied: Flask-Login==0.5.0 in /root/.virtualenvs/augur_env/lib/python3.8/site-packages (from message-insights-worker==0.0.0) (0.5.0)
    default: Requirement already satisfied: Flask-WTF==0.14.3 in /root/.virtualenvs/augur_env/lib/python3.8/site-packages (from message-insights-worker==0.0.0) (0.14.3)
    default: Requirement already satisfied: Flask==1.1.4 in /root/.virtualenvs/augur_env/lib/python3.8/site-packages (from message-insights-worker==0.0.0) (1.1.4)
    default: Collecting Keras-Preprocessing==1.1.2
    default:   Downloading Keras_Preprocessing-1.1.2-py2.py3-none-any.whl (42 kB)
    default: Collecting Keras==2.4.3
    default:   Downloading Keras-2.4.3-py2.py3-none-any.whl (36 kB)
    default: Collecting bs4==0.0.1
    default:   Downloading bs4-0.0.1.tar.gz (1.1 kB)
    default: Requirement already satisfied: click==7.1.2 in /root/.virtualenvs/augur_env/lib/python3.8/site-packages (from message-insights-worker==0.0.0) (7.1.2)
    default: Collecting emoji==1.2.0
    default:   Downloading emoji-1.2.0-py3-none-any.whl (131 kB)
    default: Collecting gensim==3.8.3
    default:   Downloading gensim-3.8.3-cp38-cp38-manylinux1_x86_64.whl (24.2 MB)
    default: Requirement already satisfied: h5py~=3.1.0 in /root/.virtualenvs/augur_env/lib/python3.8/site-packages (from message-insights-worker==0.0.0) (3.1.0)
    default: Requirement already satisfied: joblib==1.0.1 in /root/.virtualenvs/augur_env/lib/python3.8/site-packages (from message-insights-worker==0.0.0) (1.0.1)
    default: Requirement already satisfied: nltk==3.5 in /root/.virtualenvs/augur_env/lib/python3.8/site-packages (from message-insights-worker==0.0.0) (3.5)
    default: Requirement already satisfied: numpy==1.19.5 in /root/.virtualenvs/augur_env/lib/python3.8/site-packages (from message-insights-worker==0.0.0) (1.19.5)
    default: Requirement already satisfied: pandas==1.1.3 in /root/.virtualenvs/augur_env/lib/python3.8/site-packages (from message-insights-worker==0.0.0) (1.1.3)
    default: Requirement already satisfied: psycopg2-binary==2.8.6 in /root/.virtualenvs/augur_env/lib/python3.8/site-packages (from message-insights-worker==0.0.0) (2.8.6)
    default: Requirement already satisfied: requests==2.22.0 in /root/.virtualenvs/augur_env/lib/python3.8/site-packages (from message-insights-worker==0.0.0) (2.22.0)
    default: Collecting scikit-image==0.18.1
    default:   Downloading scikit_image-0.18.1-cp38-cp38-manylinux1_x86_64.whl (30.2 MB)
    default: Requirement already satisfied: scipy==1.6.3 in /root/.virtualenvs/augur_env/lib/python3.8/site-packages (from message-insights-worker==0.0.0) (1.6.3)
    default: Requirement already satisfied: sklearn==0.0 in /root/.virtualenvs/augur_env/lib/python3.8/site-packages (from message-insights-worker==0.0.0) (0.0)
    default: Collecting tensorflow==2.5.0
    default:   Downloading tensorflow-2.5.0-cp38-cp38-manylinux2010_x86_64.whl (454.4 MB)
    default: scripts/install/workers.sh: line 10:  7810 Killed                  pip install .
```

**Review by maxhbr — COMMENTED** [review]
still failing

**Comment by sgoggins** [comment]
I'm going to merge it and update a copy of our test branch. I think this may be a byproduct of some dependencies I locked down too hard.  Specifically Dask; which is changing **very** fast right now.

File: Vagrantfile (hunk 1 above)
Comment (sgoggins) [reply Vagrantfile:49]: @maxhbr: I have only occasionally used vagrant. I'm going to test on the latest available version for OSX Big Sur. Let me know if I should be testing on a different version.

**Pull request merged**